app/controllers/users_controller.rb

```diff
@@ -1,12 +1,22 @@
 class UsersController < ApplicationController
   def new
+    @user = User.new(session[:user_params])
+    session.delete(:user_params)
```

Comment:
sessions want to be remembered throughout requests. 
Sessions began at the sad path of the create method and is recalled in the 'new' method.